Homework/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>Tyrone Biggums</title>
+		<link rel="stylesheet" href="css/styles.css">
```

Comment:
@JsonLitz :hammer: You don't actually have your styles.css file in a "css" folder, which is fine. But you have to then omit the "css/" from the href to make it work.